frontend/src/features/whats-new/FeatureUpdateList.ts

```diff
@@ -16,7 +16,7 @@ export interface FeatureUpdate {
 export const FeatureUpdateList: FeatureUpdate[] = [
```

Comment:
not this PR, but we'll probably need to update this before launch. can you make an issue and add a TODO here?

Reply:
the dates, and the actual features we want to showcase. i suspect the figma text are placeholders. maybe sync with PMs/Kenneth

Reply:
Ahhh I see. I will clarify with Pearly once she is back.